index.html

```diff
@@ -301,6 +301,55 @@ <h3>
           </table>
         </section>
 
+        <section id="normative">
+          <h3>
+            Registries
+          </h3>
+          <p>
+A series of at least the following registries that support the normative
+deliverables of the Working Group will be published and maintained:
+          </p>
+          <table>
+            <tr>
+              <th>Registry</th>
+              <th>Description</th>
+            </tr>
+            <tr>
+              <td>
+<a href="https://w3c-ccg.github.io/vc-extension-registry/">Verifiable Credential Extensions Registry</a>
+              </td>
+              <td>
+A registry of extensions to the Verifiable Credentials Data Model specification.
+              </td>
+            </tr>
+            <tr>
+              <td>
+<a href="https://w3c-ccg.github.io/security-vocab/#classes">Cryptographic Container and Suite Registry</a>
+              </td>
+              <td>
+A registry of cryptographic container formats and cryptographic suites that
+are useful in conjunction with specifications managed by this Working Group
+              </td>
+            </tr>
+            <tr>
+              <td>
+<a href="https://w3c.github.io/did-spec-registries/#verification-method-types">Verification Method Registry</a>
+              </td>
+              <td>
+A registry of <a href="https://w3c-ccg.github.io/data-integrity-spec/#dfn-verification-method">verification methods</a>.
```

Comment:
I agree we might create such registry, but a WG charter should not include a registry for a CCG document. Suggest we remove this item from the charter.

Reply:
As above.

Reply:
Addressed in https://github.com/w3c/vc-wg-charter/pull/85#discussion_r817955642.